commands/toggleintake.py

```diff
@@ -1,15 +1,18 @@
 from commands2 import CommandBase
 
 from subsystems.intakesubsystem import IntakeSubsystem
+from subsystems.indexersubsystem import IndexerSubsystem
 
 
 class ToggleIntake(CommandBase):
-    def __init__(self, intake: IntakeSubsystem) -> None:
+    def __init__(self, intake: IntakeSubsystem, indexer: IndexerSubsystem) -> None:
         CommandBase.__init__(self)
         self.setName(__class__.__name__)
         self.intake = intake
+        self.indexer = indexer
         self.addRequirements(intake)
 
     def initialize(self) -> None:
         print(f"Command: {self.getName()}")
         self.intake.toggleIntake()
+        self.indexer.toggleIndexerSystem()
```

Comment:
Why does the next command "ReverseBallPath" have end() and isFinished() functions and this one does not?  Do only some commands have those?

Reply:
caused by an inconsistency of using whenHeld and whenPressed respectively, only the whenHeld needed end() and isFinished()
I will update the commands to both use whenPressed